site/src/components/TemplateStats/TemplateStats.tsx

```diff
@@ -45,6 +47,11 @@ export const TemplateStats: FC<TemplateStatsProps> = ({ template, activeVersion
           {dayjs().to(dayjs(template.updated_at))}
         </span>
       </div>
+      <div className={styles.statsDivider} />
+      <div className={styles.statItem}>
```

Comment:
Just an idea: what do you think about changing the `min-width` on the `statItem` class to be `20%`? That way, header items would expand to fill the row a bit better, i.e:
![Screen Shot 2022-06-10 at 10 41 39 AM](https://user-images.githubusercontent.com/19142439/173090247-2e830dd3-6e61-4552-b2b4-aaa0a976c808.png)

Reply:
Sounds like a great idea! Also, a step towards the design being more responsive. 🎉